migrate/20251103_update_pg_amis.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    run <<~SQL
```

Comment:
It is unsupported in Sequel to issue multiple statements in a single `run` command. It does work correctly with Sequel's postgres adapter, but you are relying on unsupported behavior. Please change this to use a separate `run` method per statement.